modules/end-user-guide/partials/proc_creating-workspaces.adoc

```diff
@@ -2,4 +2,210 @@
 [id="creating-workspaces"]
 = Creating workspaces
 
-Rough technical info to be dumped here by the assigned SME.
+If your use case does not permit use of the {prod-short} dashboard, you can create workspaces with {orch-name} APIs by applying custom resources to the cluster.
+
+[NOTE]
+====
+
+Using the {prod-short} dashboard is the default way to create workspaces because {prod-short} and its components automatically convert the target Git repository's devfile into the `DevWorkspace` and `DevWorkspaceTemplate` custom resources on the cluster with the following configuration benefits:
```

Comment:
```suggestion

Creating new workspaces through the {prod-short} dashboard provides a much better user experience compared to using the command line: When you create a workspace through the {prod-short} dashboard, you are automatically logged in to the cluster; whereas when using the command line, that does not happen, and {platforms-name} clients do not work automatically. Moreover, when you create a workspace through the {prod-short} dashboard, {prod-short} and its components automatically convert the target Git repository's devfile into the `DevWorkspace` and `DevWorkspaceTemplate` custom resources on the cluster with the following configuration benefits:

```

Reply:
I was expecting the "the developer is automatically logged in the Kubernetes cluster" expressed as one more bullet point in list of benefits of using the dashboard. The the bullet points are clear and effective, I love that. I would prefer that rather than a longer introduction but this is a minor thing so it's also good enough like that.

Reply:
In my most recent commits, I've added the bullet point:

https://github.com/eclipse-che/che-docs/blob/9175f3a1d074fe6948632cb152ac7db098373883/modules/end-user-guide/partials/proc_creating-workspaces.adoc#L12-L19